app/controllers/mentor/knowledges_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Mentor::KnowledgesController < Mentor::BaseController
+  skip_before_action :authenticate_mentor
+
+  def index
+    @knowledges = Knowledge.order("created_at desc")
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->